test/e2e/metrics_multi_pipeline_test.go

```diff
@@ -130,15 +131,19 @@ var _ = Describe(suite.ID(), Label(suite.LabelMetrics), Ordered, func() {
 				g.Expect(err).NotTo(HaveOccurred())
 
 				expectedMetrics := slices.Concat(kubeletstats.DefaultMetricsNames, k8scluster.DefaultMetricsNames)
-				g.Expect(bodycontent).To(WithFlatMetrics(WithNames(Not(ContainElements(expectedMetrics)))), "No kubeletstats metrics must be sent to prometheus backend")
-				g.Expect(bodycontent).To(WithFlatMetrics(WithScopeAndVersion(Not(ContainElement(And(HaveField("Name", InstrumentationScopeRuntime),
-					HaveField("Version",
-						SatisfyAny(
-							ContainSubstring("main"),
-							ContainSubstring("1."),
-							ContainSubstring("PR-"),
-						)),
-				))))), "scope '%v' must not be sent to the prometheus backend", InstrumentationScopeRuntime)
+				g.Expect(bodycontent).To(WithFlatMetricsDataPoints(WithAllNames(Not(ContainElements(expectedMetrics)))), "No kubeletstats metrics must be sent to prometheus backend")
```

Comment:
Sometimes it's bodycontent, sometimes bodyContent. Could you make it consistent?